sources/packages/web/src/components/common/students/StudentProfile.vue

```diff
@@ -109,6 +109,52 @@
         /></v-col>
       </v-row>
     </content-group>
+    <h3 class="category-header-medium mt-4">Legacy Match</h3>
+    <content-group v-if="studentDetail.legacyProfile">
+      <v-row>
+        <v-col
+          ><title-value
+            propertyTitle="Given names"
+            :propertyValue="studentDetail.legacyProfile.firstName"
```

Comment:
It would be good to use `emptyStringFiller` in potentially nullable properties. (In legacy it could be applicable in many fields)

Reply:
Added to both "Given names" in the UI.